spec/src/main/asciidoc/app-programming-model.adoc

```diff
@@ -388,9 +389,63 @@ public class MeteredBean {
 }
 ----
 
+==== @SimplyTimed
+An annotation for marking a constructor or method of an annotated object as simply timed.
+The metric of type SimpleTimer tracks how frequently the annotated object is invoked, and tracks how long it took the invocations to complete.
+
+The implementation must support the following annotation targets:
+
+  * `CONSTRUCTOR`
+  * `METHOD`
+  * `TYPE`
+
+If the metric no longer exists in the `MetricRegistry` when the annotated element is invoked then an `IllegalStateException` will be thrown.
+
+The following lists the behavior for each annotation target.
+
+===== CONSTRUCTOR
+
+When a constructor is annotated, the implementation must register a simple timer for the constructor using the <<annotated-naming-convention>>. Each time the constructor is invoked, the execution will be timed.
+
+.Example of an annotated constructor
+[source, java]
+----
+@SimplyTimed
+public SimplyTimedBean() {
+}
+----
+
+===== METHOD
+
+When a non-private method is annotated, the implementation must register a simple timer for the method using the <<annotated-naming-convention>>. Each time the method is invoked, the execution will be timed.
+
+.Example of an annotated method
+[source, java]
+----
+@SimplyTimed
+public void run() {
+}
+----
+
+===== TYPE
+When a type/class is annotated, the implementation must register a simple timer for each of the constructors and non-private methods using the <<annotated-naming-convention>>. Each time a constructor/method is invoked, the execution will be timed with the corresponding simple timer.
+
+.Example of an annotated type/class
+[source, java]
+----
+@SimplyTimed
+public class SimplyTimedBean {
+
+  public void simplyTimedMethod1() {}
+  public void simplyTimedMethod2() {}
+
+}
+----
+
+
 ==== @Timed
 An annotation for marking a constructor or method of an annotated object as timed.
-The metric of type Timer tracks how frequently the annotated object is invoked, and tracks how long it took the invocations to complete.
+The metric of type Timer tracks how frequently the annotated object is invoked, and tracks how long it took the invocations to complete. The data is aggregated to calculate duration statistics and throughput statistics.
```

Comment:
same here, let's avoid "how frequently" which suggests it is something like a meter